test/test_apps/shared/test/page_objects/books/dummy_show_page_with_unresolvable_path.rb

```diff
@@ -0,0 +1,11 @@
+module PageObjects
+  module Books
+    class DummyShowPageWithUnresolvablePath < AePageObjects::Document
+      # This page is used to test when the first path is correct, and the second path is incorrect.
+      # In the past, the code will blow up with an unreadable error that doesn't indicate that the
+      # failure is due to the PathNotResolvable. We now tests that it has a understandable error message.
+      path :book
```

Comment:
The first path seems to be special, and is used by the visit method to generate the path to visit.
Therefore, when doing `Document.visit`, [it throws](https://github.com/appfolio/ae_page_objects/blob/master/lib/ae_page_objects/document.rb#L21) PathNotResolvable if the first path wasn't valid.